website/content/labs/lab4.md

```diff
@@ -86,7 +85,7 @@ As you are stepping through your program, you might want to be able to see the v
 
 There is also a useful command to keep track of when a variable changes. This command is `watch`. If you want to keep track of when the variable `x` changes, type `watch x`. Every time it is changed when you run your program or execute a line of code, the program will stop and print the old and new values.
 
-Now that you know these commands, let’s have some practice with them! Using gdb, answer the questions in `task2.txt` about `task2.c`.
+Now that you know these commands, let’s have some practice with them! Using gdb, answer the following questions on Gradescope:
 
 1. At what line(s) does the value of variable `c` change?
```

Comment:
Does the initialization of c count as a change?

Reply:
The initialization of c counts as a change; I just double-check that gdb catches this if you "watch c" from the start of the program (and mentioned that you can use "break main" to break right when the program starts)

Reply:
I ended up adding some extra text to explain next separately from step, with a concrete example that highlights that next runs the line that is being displayed (and also clarified that you can just press enter to keep stepping through the lines)